src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/ConsignmentRepository.scala

```diff
@@ -42,7 +42,7 @@ class ConsignmentRepository(db: Database, timeSource: TimeSource) {
   }
```

Comment:
There is an unused import (line 7): `import slick.sql.SqlStreamingAction`

Reply:
Ah I'll remove that :+1: